content/doc/book/pipeline/syntax.adoc

```diff
@@ -257,33 +257,31 @@ For example, if you want a pod with a Kaniko container inside it, you would defi
 ----
 agent {
     kubernetes {
-        label podlabel
-        yaml """
+        defaultContainer 'kaniko'
+        yaml '''
 kind: Pod
-metadata:
-  name: jenkins-agent
 spec:
   containers:
   - name: kaniko
     image: gcr.io/kaniko-project/executor:debug
     imagePullPolicy: Always
     command:
-    - /busybox/cat
-    tty: true
+    - sleep
+    args:
+    - 99d
```

Comment:
indent looks broken here
```suggestion
      args:
        - 99d
```

Reply:
https://github.com/jenkins-infra/jenkins.io/blob/7f8983e9ce53ab7709e069fb32e8e6a6cfe78d0b/content/doc/book/pipeline/syntax.adoc?plain=1#L264-L272 looks right to me, WDYM? `args` is an attribute of the container, with a value consisting of an array of one string; `command` is its sibling attribute.

Reply:
oops my bad, mis-read, looks good and checked the preview site